.github/workflows/build.yml

```diff
@@ -118,7 +118,9 @@ jobs:
       - name: run pytest
         run: |
           pip install pytest pytest-cov pytest-xdist
-          pytest -n2 --dist=loadfile
+          cd /tmp
+          pytest -n2 --dist=loadfile --pyargs weldx
+          cd -
```

Comment:
can you share why this is necessary?

Reply:
https://github.com/BAMWelDX/weldx/runs/2475826648#step:62 Because of this error. Somehow, if we run the tests from a dist install within the source directory, an import conflict appears. Running the tests elsewhere cures this.